test/es7/PromiseRejectionTracking.js

```diff
@@ -0,0 +1,145 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Test HostPromiseRejectionTracker - see ecma262 section 25.4.1.9
+
+let tests = [
+	{
+		name: "Reject promise with no reactions.",
```

Comment:
super nit: looks like you used tabs here, can you switch over to spaces to match the majority of the files in the same directory?

Reply:
Sorry about that, fixed.